package.json

```diff
@@ -59,10 +61,14 @@
     "decentraland-ui2": "^0.0.3-9857431499.commit-4197294",
     "electron-updater": "6.2.1",
     "flat": "^6.0.1",
+    "is-running": "^2.1.0",
+    "npm": "^10.8.1",
```

Comment:
Electron has embed a node version right ? Or how does this work ? 
If the answer is yes, how are we sure that this npm version is compatible with the node version installed ? Do we need to validate ?
Not sure how this works, if i'm saying non-sense things just ignore me 🙃

Reply:
yes, potentially if the npm version stays too much behind the node version, they could be incompatible. I think the best approach would be to update the `npm` version whenever we upgrade major node versions (like when we jump to the next LTS node version). 

In short, once we update the `electron` package we should also upgrade the `npm` one to make sure they will be in sync. This shouldn't be something that happens too often though.

Reply:
we could eventually add a build check for this if necessary. At most, if we want to run on supported versions, we would be doing this once a year...

https://www.electronjs.org/docs/latest/tutorial/electron-timelines